.circleci/config.yml

```diff
@@ -20,6 +21,9 @@ parameters:
     type: enum
     enum: ["stable", "beta", "unstable"]
     default: "stable"
+  releaseCommand:
+    type: string
+    default: "foo" # when specified this mean the workflow was called from release-tool.
```

Comment:
Why "foo"? Maybe this could be something that reflects state in the console. Something like "echo No release_command specified"

Reply:
good point , I will update along with variable change when https://github.com/twilio/release-tool/pull/3 merges.